packages/palette/src/elements/Carousel/Carousel.tsx

```diff
@@ -8,18 +8,54 @@ import React, {
   useState,
 } from "react"
 import styled from "styled-components"
+import { system } from "styled-system"
 import { useCursor } from "use-cursor"
 import { ChevronIcon } from "../../svgs"
 import { SpacingUnit } from "../../Theme"
 import { useUpdateEffect } from "../../utils/useUpdateEffect"
 import { Box, BoxProps } from "../Box"
 import { Skip } from "../Skip"
 import { VisuallyHidden } from "../VisuallyHidden"
-import { CarouselNext, CarouselPrevious } from "./CarouselNavigation"
+import {
+  CarouselNavigationProps,
+  CarouselNext,
+  CarouselPrevious,
+} from "./CarouselNavigation"
 import { paginateCarousel } from "./paginate"
 
 const RAIL_TRANSITION_MS = 500
 
+const transition = system({ transition: true })
+
+/** CarouselRailProps */
+export type CarouselRailProps = BoxProps & { transition?: string }
```

Comment:
should be use the return type of `transition` vs hardcode?

Reply:
Unfortunately the return type is `styleFn` which is a higher order function yada yada. The most correct thing is probably to install csstype and use `ResponsiveValue<Property.Transition>` but that's gonna be string regardless. I did update it to be `ResponsiveValue<string>` though 👍